### DISCUSSION TITLE
Welcome to koding Discussions!

### DISCUSSION THREAD

**Discussion by hilmanski** [hilmanski/koding#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by perdiDev** [comment]
Hi! Perkenalkan saya perdi, web developer indonesia yang sedang menempuh pendidikan di Universitas Hasanuddin. 

Saya mau berkontribusi untuk pengembangan proyek ini agar saya bisa memiliki lingkungan programing yang kompetitif. Kompetitif dalam hal yang positif. Dengan proyek ini saya berharap semakin banyak developer yang mungkin akan termudahkan untuk mempelajari programing, terutamanya web development. Oleh karenanya saya ingin berkontribusi  👋

**Reply by hilmanski** [reply]
terima kasih senang banyak yang ingin berkontribusi. salam kenal
